app/css/main.css

```diff
@@ -1,48 +1,56 @@
-/* Tagname styles */
-body {
-  margin: 0;
-  padding: 0;
-  font-family: 'Google Sans', 'Roboto', 'Noto', sans-serif;
-  -webkit-font-smoothing: antialiased;
-  color: #222;
-  line-height: 1.5;
-}
-
+/* Theme colors */
 a {
-  text-decoration: none;
+  color: #FF4470;
 }
-
 h1 {
   font-size: 40px;
   line-height: 48px;
   font-weight: normal;
 }
-
 h2 {
   font-size: 28px;
   line-height: 36px;
   font-weight: normal;
 }
-
 h3 {
   font-size: 20px;
   line-height: 30px;
   font-weight: normal;
 }
 
+ol li,
+ul li {
+  padding-left: 12px;
```

Comment:
Intended across all sites?

Reply:
Yes, we need bigger left padding for list items.